kwok/cloudprovider/register.go

```diff
@@ -0,0 +1,45 @@
+/*
+Copyright The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kwok
+
+const (
+	Group              = "karpenter.kwok.sh"
```

Comment:
Kwok is currently using `kwok.x-k8s.io`. Maybe worth a question to their team if we can extend off of it to use `karpenter.kwok.x-k8s.io`. This doesn't require formal approval from API since it's `x-k8s.io` but it seems odd that we would use `.sh` here when other `kwok` resources (and the website that they host themselves) doesn't use the `.sh` domain suffix

Reply:
I can send a question to the maintainers over there to see what their thought is, but think we can change that really at any point.

Reply:
Sure, "all of these labels are alpha", "this is really only meant for dev and we can break you at any time"